spec/commands/decidim/proposals/endorse_proposal_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Decidim
+  module Proposals
+    describe EndorseProposal do
+      let!(:component) { create(:proposal_component) }
+      let!(:scope) { create(:scope, organization: component.organization) }
+      let!(:other_scope) { create(:scope, organization: component.organization) }
+      let!(:user) { create(:user, :confirmed, organization: component.organization) }
+      let!(:user_group) { create(:user_group, :confirmed, :verified, organization: component.organization, users: [user]) }
+      let!(:proposal) { create(:proposal, component: component) }
+      let!(:emendation) { create(:proposal, component: component) }
+      let!(:amendment) { create(:amendment, amendable: proposal, emendation: emendation) }
+
+      let(:command) { described_class.new(resource, user, user_group_id) }
+
+      describe "User endorses Proposal resource" do
```

Comment:
why do we use the "resource" terminology in the whole spec if this command is only for Proposals?
```suggestion
      describe "User endorses Proposal" do
```

Reply:
Mainly for this two contexts:

> context "when the resource is an orginial proposal" do
> context "when the resource is an amendment" do

Should they say the folllowing?

> context "when the proposal is an amendable" do
> context "when the proposal is an emendation" do